portals/application/configs/klear/model/Terminals.yaml

```diff
@@ -31,10 +31,6 @@ production:
         icon: help
         text: _("Allowed characters: a-z, A-Z, 0-9, underscore and '*'")
         label: _("Need help?")
-    domain:
-      title: _('Domain')
-      type: text
-      maxLength: 255
     disallow:
```

Comment:
Domain is a field in Terminals list screen, can we remove the entire field configuration?

Reply:
In list but not in model if I'm not wrong. Friends for example does not have the domain field already. The domain is inherited from company but we don't fill that field on terminals creation.

Reply:
In fact it could be totally dropped from database. As you can get that value from the company itself. But I understood that was by design for some performance reason. Otherwise it would be a lot better to drop domain from Terminals, Friends and Retail Accounts.

Reply:
Afaik fields are there because it's easier for kamailio to query that tables directly when checking credentials. It's a bit PITA because when a domain changes we have to update a lot of places..

Although not used in the edit screen, Domain field is present in the list screen and the text from the header of the table is taken from the model (Domain for english, Dominio for spanish). We could remove maxLength, I think that's not used at all for listing.

Reply:
Understood. I insert it again. I include maxLength too with value 190 just in case.